### ISSUE TITLE
Problem with Safari 11.00

### ISSUE THREAD

**Issue by giovaANTO** [matthieua/WOW#303]
Hello, since i've upload my browser to safari 11.00 i've seen that all the elements that have wow  + animation class doesn't show up. 

This bug shows up only in the safari browser, in the others ( like Chrome, Mozilla, Opera etc..) the problem doesn't persists. 

If i remove the classes the item are displayed normally.

code works well in safari < 11.00 (like 10.12).

Someone have the same situation?
Thanks in advance for all the help

**Comment by hardenrahul** [comment]
yes i have the same issue please someone get it solve as soon as possible.

**Comment by ststaynov** [comment]
+1

**Comment by kaganvmz** [comment]
I have too +1
